scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -50,7 +50,7 @@ apt_arm_packages:
 
 validation_distribution_binaries:
   - "/usr/bin/shellcheck"
-  - "/usr/bin/clang-7"
+  - "/usr/bin/clang-8"
   - "/usr/bin/clang-8"
```

Comment:
You have a couple of double dependencies in the above files. Clang8 was added before, so only the lvi wrapper change is needed and then deprecating clang-7 which can be seen as optional.

Reply:
51/52 are the same.